### PR TITLE
Add flags for websocket read/write deadlines

### PR DESCRIPTION
**Pull request by aidan-kwon**
## Proposed changes

- Add flags for websocket read/write deadlines

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: networks/rpc/websocket.go [hunk 1]
```diff
@@ -98,6 +104,13 @@ func (srv *Server) FastWebsocketHandler(ctx *fasthttp.RequestCtx) {
 	err := upgrader.Upgrade(ctx, func(conn *fastws.Conn) {
 		//Create a custom encode/decode pair to enforce payload size and number encoding
 		encoder := func(v interface{}) error {
+			if WebsocketReadDeadline != 0 {
+				conn.SetReadDeadline(time.Now().Add(time.Duration(WebsocketReadDeadline) * time.Second))
+			}
+			if WebsocketWriteDeadline != 0 {
+				conn.SetWriteDeadline(time.Now().Add(time.Duration(WebsocketWriteDeadline) * time.Second))
+			}
+
```
Comment (jeongkyun-oh) [inline networks/rpc/websocket.go:113]: Shouldn't it be in `Upgrade`?

Comment (aidan-kwon) [reply networks/rpc/websocket.go:113]: My mistake. Thanks you!

**Review by jeongkyun-oh — APPROVED** [review]

File: networks/rpc/websocket.go [hunk 2]
```diff
@@ -70,6 +70,12 @@ func (srv *Server) WebsocketHandler(allowedOrigins []string) http.Handler {
 		Handler: func(conn *websocket.Conn) {
 			// Create a custom encode/decode pair to enforce payload size and number encoding
 			conn.MaxPayloadBytes = common.MaxRequestContentLength
+			if WebsocketReadDeadline != 0 {
```
Comment (winnie-byun) [inline networks/rpc/websocket.go:73]: The zero value is handled in `conn.SetReadDeadline` function. We don't need to check if the value is zero.

**Review by winnie-byun — APPROVED** [review]

File: networks/rpc/websocket.go (hunk 2 above)
Comment (aidan-kwon) [reply networks/rpc/websocket.go:73]: @winnie-byun There is an addition in the next line. When `WebsocketReadDeadline == 0`, the input parameter is not zero. The next line will be the same with `conn.SetReadDeadline(time.Now())` that means "expire right now".

Comment (winnie-byun) [reply networks/rpc/websocket.go:73]: I am confused with the code. If I set `WebsocketReadDeadline` to 10 seconds, does it mean I can only use the websocket for next 10 seconds after the node started?

Comment (aidan-kwon) [reply networks/rpc/websocket.go:73]: No, the deadline is set when each time the handler executed. It means that each client has its own deadline when the connection is established.

**Review by yoomee1313 — APPROVED** [review]

**Comment by aidan-kwon** [comment]
@KimKyungup  I'm gonna merge this PR. If you have any comments, please leave them here later.

**Pull request merged**